### PR TITLE
Enable deploying to multiple environments and cleanup old passcode fu…

### PR DESCRIPTION
**Pull request by brentwalter**
This work adds an extra parameter to our deployment script to specify an --environment parameter that can be either "dev" or "prod".

I also cleaned up some of the old Twilio application passcode functionality so that folks wouldn't be confused going forward.

There will be a complimentary pull request on the With repo to utilize this functionality.

### CODE REVIEW

File: src/helpers.js [hunk 1]
```diff
@@ -77,19 +77,20 @@ async function getAppInfo() {
 
   const assets = await appInstance.assets.list();
 
-  const passcodeVar = variables.find(v => v.key === 'API_PASSCODE');
-  const expiryVar = variables.find(v => v.key === 'API_PASSCODE_EXPIRY');
+  // const passcodeVar = variables.find(v => v.key === 'API_PASSCODE');
```
Comment (brentwalter) [inline src/helpers.js]: I kept a bunch of this stuff commented out and didn't delete it until I've been able to do enough rounds of production testing to be sure I didn't make the entire world collapse.

File: src/commands/rtc/apps/video/deploy.js [hunk 2]
```diff
@@ -68,6 +74,13 @@ DeployCommand.examples = [
   `# Deploy an application token server with passcode authentication
 $ twilio rtc:apps:video:deploy --authentication passcode
 deploying app... done
+Passcode: 1111111111`,
+  `
+# Deploy an application token server to production environment
+# There are two environments: "dev" and "prod". "dev" is default.
+# This is an optional flag. Defualts to "dev", development environment
```
Comment (wpultz) [inline src/commands/rtc/apps/video/deploy.js]: Typo *Defaults

File: src/helpers.js [hunk 3]
```diff
@@ -105,21 +94,22 @@ async function displayAppInfo() {
   if (appInfo.hasAssets) {
     console.log(`Web App URL: ${appInfo.url}`);
   }
-  console.log(`Passcode: ${appInfo.passcode}`);
-  console.log(`Expires: ${appInfo.expiry}`);
+  console.log(`App SID: ${appInfo.sid}`);
+  // console.log(`Passcode: ${appInfo.passcode}`);
```
Comment (wpultz) [inline src/helpers.js]: Leaving these for posterity?

File: src/helpers.js [hunk 4]
```diff
@@ -128,11 +118,11 @@ async function deploy() {
     env: {
       TWILIO_API_KEY_SID: this.twilioClient.username,
       TWILIO_API_KEY_SECRET: this.twilioClient.password,
-      API_PASSCODE: pin,
-      API_PASSCODE_EXPIRY: expiryTime,
+      // API_PASSCODE: pin,
```
Comment (wpultz) [inline src/helpers.js]: Leaving these for posterity?

File: src/commands/rtc/apps/video/deploy.js (hunk 2 above)
Comment (brentwalter) [reply src/commands/rtc/apps/video/deploy.js]: will fix

File: src/helpers.js (hunk 3 above)
Comment (brentwalter) [reply src/helpers.js]: nope, gone

File: src/helpers.js (hunk 4 above)
Comment (brentwalter) [reply src/helpers.js]: nope, gone

**Review by wpultz — APPROVED** [review]

**Pull request merged**